t/venom/pfservers/backup_db_and_restore/00_backup_db_and_restore.yml

```diff
@@ -0,0 +1,78 @@
+name: Backup db and restore 
+version: 2
+testcases:
+- name: create_users
+  steps:
+    - type: http
+      method: POST
+      url: '{{.pfserver_webadmin_url}}/api/v1/users'
+      ignore_verify_ssl: true
+      body: >-
+        {
+          "pid_overwrite": "0",
+          "pid": "{{.backup_db_and_restore.create_user}}",
+          "sponsor": "admin",
+          "valid_from": "{{.backup_db_and_restore.registration_user}}",
+          "expiration": "{{.backup_db_and_restore.expiration_user}}",
+          "actions": [
+            {
+              "type": "set_access_level",
+              "value": "NONE"
+            }
+          ]
+        }
+      headers:
+        "Authorization": "{{.pfserver_token}}"
+        "Content-Type": "application/json"
+      assertions:
+        - result.statuscode ShouldEqual 201
+
+- name: Backup
+  steps:
+  - script: /usr/local/pf/addons/backup-and-maintenance.sh
+
+- name: get_backup_name
+  steps:
+  - script: 'find /root/backup/packetfence-db-dump-*.sql.gz -mmin -5'
+    vars: 
+      backup_name:
+        from: result.systemout
+
+- name: unzip_db_backup
+  steps:
+  - script: 'gunzip {{.get_backup_name.backup_name}}'
+
+- name: remove_db_pf
+  steps:
+  - script: mysql -u {{.backup_db_and_restore.db_user}} -p{{.backup_db_and_restore.password_db_user}} -e "drop database pf;"
+
+- name: create_db_pf
+  steps:
+  - script: mysql -u {{.backup_db_and_restore.db_user}} -p{{.backup_db_and_restore.password_db_user}} -e "create database pf;"
+    assertions:
+    - result.code ShouldEqual 0
+
+- name: get_backup_name_uncompressed
+  steps:
+  - script: 'find /root/backup/packetfence-db-dump-*.sql -mmin -5'
```

Comment:
No need to re-run a find, you already have filename in `{{.get_backup_name.backup_name}}`. You can use `basename {{.get_backup_name.backup_name}} .gz` but you will get only filename (no full path).